contrib/snowball/NEWS

```diff
@@ -47,7 +47,7 @@ New Code Generators
 * Add Python generator (#24).  Originally written by Yoshiki Shibukawa, with
   additional updates by Dmitry Shachnev.
 
-* Add Javascript generator.  Based on JSX generator (#26) written by Yoshiki
+* Add JavaScript generator.  Based on JSX generator (#26) written by Yoshiki
```

Comment:
Brand (but probably an upstream change that should be sent somewhere -- if someone identifies the upstream, I can queue working on it)

Reply:
Upstream is here: https://github.com/snowballstem/snowball